### DISCUSSION TITLE
[V5] I want call get file css

### DISCUSSION THREAD

**Discussion by b23618** [adonisjs/core#2613] [Help]
Hello i want call file css but error
error code : 
 `{"message":"E_ROUTE_NOT_FOUND: Cannot GET:/main.css","stack":"HttpException: E_ROUTE_NOT_FOUND: Cannot GET:/main.css\n    at Function.invoke (C:\\ruk-ruay\\lotterythai\\server\\node_modules\\@adonisjs\\http-server\\build\\src\\Exceptions\\HttpException.js:30:23)\n    at RequestHandler.findRoute (C:\\ruk-ruay\\lotterythai\\server\\node_modules\\@adonisjs\\http-server\\build\\src\\Server\\RequestHandler\\index.js:49:49)\n    at RequestHandler.handle (C:\\ruk-ruay\\lotterythai\\server\\node_modules\\@adonisjs\\http-server\\build\\src\\Server\\RequestHandler\\index.js:65:14)\n    at C:\\ruk-ruay\\lotterythai\\server\\node_modules\\@adonisjs\\http-server\\build\\src\\Server\\index.js:75:44\n    at processTicksAndRejections (internal/process/task_queues.js:93:5)\n    at Server.handle (C:\\ruk-ruay\\lotterythai\\server\\node_modules\\@adonisjs\\http-server\\build\\src\\Server\\index.js:136:13)","code":"E_ROUTE_NOT_FOUND"}`

Help me please!!!

link: http://localhost:3333/main.css

**Comment by thetutlage** [comment]
In which directory the css file is stored?

**Reply by b23618** [reply]
Css file in directory public floder (root project)

**Reply by thetutlage** [reply]
Have you enabled static assets inside the config file? https://docs.adonisjs.com/guides/static-assets#configuration
